sema4ai/vscode-client/src/activities.ts

```diff
@@ -185,6 +185,29 @@ export async function resolveInterpreter(targetRobot: string): Promise<ActionRes
     }
 }
 
+async function flattenRobotsAndActions(robotsInfo: LocalPackageMetadataInfo[]): Promise<LocalPackageMetadataInfo[]> {
+    const flattenedRobotsInfo: LocalPackageMetadataInfo[] = [];
+
+    robotsInfo.forEach((entry) => {
+        flattenedRobotsInfo.push(entry);
+
+        if (isAgentPackage(entry) && entry.organizations) {
+            entry.organizations.forEach((org) => {
+                if (org.name === "MyActions") {
```

Comment:
Maybe it should check all orgs that are not `org.name.lower().replace('.', '') == sema4ai`?

Also, shouldn't this be an option? i.e.: why can't the user run an action from `Sema4.ai` if he dowloaded it?